alembic/versions/0210dc9b2f44_migration_message.py

```diff
@@ -0,0 +1,28 @@
+"""Add a resolved column to the responses table
+
+Revision ID: 0210dc9b2f44
+Revises: b51ed27a84de
+Create Date: 2019-03-26 20:04:43.331293
+
+"""
+from alembic import op
+import sqlalchemy as sa
+
+
+# revision identifiers, used by Alembic.
+revision = '0210dc9b2f44'
+down_revision = '4b0dfa7815d5'
+branch_labels = None
+depends_on = None
+
+
+def upgrade():
+    # ### commands auto generated by Alembic - please adjust! ###
+    op.add_column('responses', sa.Column('resolved', sa.Boolean(), nullable=True))
```

Comment:
@nakatuddesuzan just a suggestion, could we have a default value for every response created to be `false` so that when it is resolved it's updated to `true`. that would mean this field will be `nullable=False` so we don't have null values for the `resolved` field of already existing responses.

Reply:
Hello @dnuwa , That s how the implementation is currently 
<img width="393" alt="Screen Shot 2019-03-29 at 11 05 23" src="https://user-images.githubusercontent.com/39129473/55218277-af924b00-5212-11e9-91a1-4c1c946ccf0e.png">

Reply:
This does not take care of already existing responses.
@nakatuddesuzan we could do something like` op.add_column('responses', sa.Column('resolved', sa. Boolean(), nullable=False, server_default=False))`
what this will do is to populate the `resolved` column of already existing responses with a value of `False` which can later be updated.

Reply:
Hey @dnuwa , do I need to just hardcode this?

Reply:
yes @nakatuddesuzan